packages/babel-plugin-transform-runtime/scripts/build-dist.js

```diff
@@ -0,0 +1,158 @@
+"use strict";
```

Comment:
The transform and the build script share the same source of data, which was my motivation. If we split it out, the build script will still have to reach into the transform to get that data, and doing that felt equally awkward.

Reply:
In that case - feel free to go with whatever u prefer, I would still prefer to have this top-level, but I'm not super attached to it now (because of the reasons u gave)

Reply:
I guess we should explain somewhere about running this